components/Welcome/index.js

```diff
@@ -22,28 +24,103 @@ const StyledPararaph = styled.p`
   text-align: center;
   opacity: 0.8;
 `;
+const StyledLink = styled(Link)`
+  color: white;
+`;
+const StyledLoginText = styled.button`
+  background: none;
+  border: none;
+  padding: 0;
+  font-size: inherit;
+  cursor: pointer;
```

Comment:
Since your app is optimized for mobile, setting the cursor isn't really necessary. But it's also no problem.

Reply:
thanks